pandas/core/frame.py

```diff
@@ -1592,11 +1592,11 @@ def from_csv(cls, path, header=0, sep=',', index_col=0, parse_dates=True,
                       "for from_csv when changing your function calls",
                       FutureWarning, stacklevel=2)
 
-        from pandas.io.parsers import read_table
-        return read_table(path, header=header, sep=sep,
-                          parse_dates=parse_dates, index_col=index_col,
-                          encoding=encoding, tupleize_cols=tupleize_cols,
-                          infer_datetime_format=infer_datetime_format)
+        from pandas.io.parsers import read_csv
+        return read_csv(path, header=header, sep=sep,
+                        parse_dates=parse_dates, index_col=index_col,
+                        encoding=encoding, tupleize_cols=tupleize_cols,
+                        infer_datetime_format=infer_datetime_format)
```

Comment:
As it stands, this is a breaking API change.

@jreback : `0.21.0` deprecations are fair game to remove in `0.24.0`, no?

Reply:
As far as I know, the only difference between `read_csv` and `read_table` is the default value of `sep`. Here, `sep` is explicitly set, so I don't see how this changes anything?

Reply:
You're right: for the vast majority, it's not going to be breaking.  It's only in a corner case situation (e.g. you pass in `None`) that you will see a change.  This is fine then.